puppet/manifests/nubots.pp

```diff
@@ -307,6 +529,108 @@
       before  => [ Installer['aravis'], ],
     }
 
+    file { "${prefix}/${arch}/src/dart.patch":
```

Comment:
Why are we patching stuff in our puppet file?

Reply:
dont get me started